app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -505,6 +505,7 @@ class Result extends React.Component {
           marking_state={this.state.marking_state}
           num_assigned={this.state.num_assigned}
           num_marked={this.state.num_marked}
+          num_collectd={this.props.num_collected}
```

Comment:
- this prop is not used
- a prop named `num_collected` is already being passed to the `SubmissionSelector` on line 499
- `num_collected` should be accessed from the `this.state`, not `this.props` (see other comment below)